docs/insomnia/introduction-to-plugins.md

```diff
@@ -91,23 +91,55 @@ Example: Plugin package.json file
+const jsonObjToBuffer = obj => Buffer.from(JSON.stringify(obj), 'utf-8');
+
+module.exports.responseHooks = [
+  async ctx => {
+    try{
+      const resp = bufferToJsonObj(ctx.response.getBody());
+      
+      // Modify
+      resp.__randomNumber = Math.random();
+
+      // If you want something from a user, use a prompt:
+      const promptResult = await ctx.app.prompt('Type something', { defaultValue: 'abcd' });
+      resp.__customValue = promptResult;
+
+      ctx.response.setBody(jsonObjToBuffer(resp));
+    } catch {
+      // no-op
+    }
+  }
+]
+```
+_This example specifically deals with making timestamps human readable. Update the functionality as needed._
+
 ## Manage Plugins
-Plugins can be added from the Plugins tab within the application preferences dialog. They can also be downloaded and installed directly from npm.
+
+Plugins can be added from the **Plugins** tab in the **Preferences** menu. They can also be downloaded and installed directly from [NPM](https://insomnia.rest/plugins).
 
 ## Publish Plugins
-Before you publish your plugin, there are two main things you want to ensure if you want your plugin to be recognized by Insomnia and be available through the Insomnia Plugin Hub.
 
-* Your plugin's package.json contains the insomnia attribute with the correct structure. See Create a Plugin for more info.
-* Your plugin's package name is prefixed with `insomnia-plugin-`
+Before you publish your plugin, ensure you have met the following criteria for your plugin to be recognized by Insomnia and be available on the [Insomnia Plugin Hub](https://insomnia.rest/plugins).
+
+* Your plugin's `package.json` contains the `insomnia` attribute with the correct structure. See [Create a Plugin](#create-a-plugin) for more info.
+* Your plugin's package name is prefixed with `insomnia-plugin-`.
+
+After you have verified that your plugin meets the criteria described above, publish your plugin following the [NPM instructions](https://docs.npmjs.com/creating-and-publishing-scoped-public-packages).
 
-After you have verified that your plugin meets the criteria described above, now you can proceed with publishing your plugin following the npm instructions on publishing a package.
+If your package does not show up on the Insomnia Plugin Hub after a few days, please [contact us](https://insomnia.rest/support) with the name of your plugin and a link to the published NPM package.
 
-If your package does not show up on the Insomnia Plugin Hub after a few days, please contact us with the name of your plugin and a link to the published npm package.
+## Debug Using Chrome
 
-## Debugging
-Navigate to View > Toggle DevTools to open the Chrome Developer Tools. From here, you can debug Insomnia as you would any web project in Chrome.
+To open the Chrome Dev tools for debugging on Chrome, right click on your mouse or trackpad and click **Inspect**. If you don't use a mouse or trackpad, click on **View**, then **Developer**, and then **Developer Tools**. 
```

Comment:
The right click won't work in production. They'll need to go through `View > Toggle DevTools`.

![image](https://user-images.githubusercontent.com/4312346/132838376-420f29af-518c-4471-8d2d-03e9ad14e5f8.png)

Reply:
The confusing came from no context of where you are when trying to access this tool. From the screenshot, it looks like you're opening debugging from the Insomnia app itself. Will update.